src/components/Collective/CollectiveActions.vue

```diff
@@ -166,6 +167,10 @@ export default {
 				? generateUrl(`/apps/collectives/p/${this.shareTokenParam}/print/${this.collective.name}`)
 				: generateUrl(`/apps/collectives/_/print/${this.collective.name}`)
 		},
+
+		unshareIcon() {
```

Comment:
Seems like a leftover or copy/paste error as well. `unshareIcon()` is not used here.